pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
     {name = "Anarchy Team", email = "matt@anarchy.ai"},
 ]
 
-requires-python = ">=3.8" # this could be lower if need be
+requires-python = ">=3.10" # this could be lower if need be
```

Comment:
yus! (what necessitates it , but i support it)

Reply:
outlines requires a higher python version